pkg/config/default_windows.go

```diff
@@ -0,0 +1,41 @@
+//go:build windows && !brew && !archivematica && !js
+// +build windows,!brew,!archivematica,!js
+
+// Copyright 2014 Richard Lehane. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"os"
+	"path/filepath"
+)
```

Comment:
It looks like there is a missing "windows" import here, it looks like it is used for `KnownFolderPath` and the other flags below.